flink-table/flink-table-api-java-bridge/src/test/java/org/apache/flink/table/api/bridge/java/internal/StreamTableEnvironmentImplTest.java

```diff
@@ -63,7 +63,7 @@ public void testAppendStreamDoesNotOverwriteTableConfig() {
 			equalTo(minRetention.toMilliseconds()));
 		assertThat(
 			tEnv.getConfig().getMaxIdleStateRetentionTime(),
-			equalTo(maxRetention.toMilliseconds()));
+			equalTo(minRetention.toMilliseconds() * 3 / 2));
```

Comment:
Please use the new method `setIdleStateRetentionTime` to set idle state in tests.